### ISSUE TITLE
Code syntax highlighting colors: poor color contrast

### ISSUE THREAD

**Issue by joedolson** [wpaccessibility/wp-a11y-docs#245]
Many of the colors used to highlight code syntax create color contrast failures. E.g., variables, comments, etc.

It's particularly bad in many of the yellow-tinged code classes (e.g. .nv, .na, .mi, .s). 

Opening a PR to fix those.

**Comment by joedolson** [comment]
Closed by https://github.com/wpaccessibility/wp-a11y-docs/pull/246
